bin/check_drupal

```diff
@@ -548,8 +552,6 @@ print_help() {
 	printf "  -n <name>              [optional] Specify a name for the drupal instance to\n"
 	printf "                         appear on the nagios output. The default is 'Drupal'\n\n"
 
-	printf "                         the 'drupal' folder. This parameter is required.\n\n"
```

Comment:
Ah, yes, my apologies, didn't read thoroughly enough :)
My OCD side says this should be a separate PR. My honest side says I'd do it in here too :D